src/sbt-test/sbt-ensime/scalariform-only/project/plugins.sbt

```diff
@@ -1 +0,0 @@
-../../scalariform/project/plugins.sbt
```

Comment:
Previous, the file was a symbolic link to the `plugins.sbt` in `scalariform` test but that test was no longer needed and I deleted it, so moved the file here